Weather App/Model/CurrentWeatherResponseModel.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  LocationModels.swift
+//  Weather App
+//
+//  Created by Shikhar Sharma on 14/03/22.
+//
+
+import Foundation
+
+public struct CurrentWeatherResponse : Codable {
```

Comment:
What is the purpose of public here and everywhere, this should be internal right?

Reply:
done